source/grid/grid_tools.cc

```diff
@@ -693,7 +692,7 @@ namespace GridTools
     DoFHandler<dim> dof_handler(triangulation);
     dof_handler.distribute_dofs(q1);
 
-    CompressedSparsityPattern c_sparsity_pattern (dof_handler.n_dofs (),
+    DynamicSparsityPattern c_sparsity_pattern (dof_handler.n_dofs (),
                                                   dof_handler.n_dofs ());
```

Comment:
Indentation. You probably need to let the indentation script run over everything.